Frameworks/UIKit/UIScrollView.mm

```diff
@@ -250,6 +291,14 @@ - (void)_initScrollViewer:(WXFrameworkElement*)xamlElement {
     _scrollViewer.maxZoomFactor = 1.0f;
     _scrollViewer.minZoomFactor = 1.0f;
     _scrollViewer.zoomMode = WXCZoomModeDisabled;
+
+    // setting transparent background so we can receive touch input
+    WUColor* convertedColor = ConvertUIColorToWUColor([UIColor clearColor]);
+    WUXMSolidColorBrush* brush = [WUXMSolidColorBrush makeInstanceWithColor:convertedColor];
```

Comment:
like that

---
In reply to: [86440003](https://github.com/Microsoft/WinObjC/pull/1305#discussion_r86440003) [](ancestors = 86440003,86439907)